frame/pablo/rpc/src/lib.rs

```diff
@@ -28,6 +28,15 @@ where
 	) -> RpcResult<
 		PriceAggregate<SafeRpcWrapper<PoolId>, SafeRpcWrapper<AssetId>, SafeRpcWrapper<Balance>>,
 	>;
+
+	#[rpc(name = "pablo_lpTokensReceivedForLiquidityProvided")]
```

Comment:
Nit:
```suggestion
	#[rpc(name = "pablo_expectedLpTokensGivenLiquidity")]
```

Reply:
would be nice making words in trait be same as in rpc ;)

Reply:
This is a much better name, i'll rename the function as well :+1: